docs/page-objects/inputs.rst

```diff
@@ -81,6 +86,11 @@ Working with HttpResponse
 
 :class:`~.HttpResponse` has many attributes and methods.
 
+.. tip:: You can use :class:`~web_poet.pages.WebPage` instead of
+         :class:`~web_poet.pages.ItemPage` to have
+         :class:`~web_poet.page_inputs.http.HttpResponse` as input and get
+         convenient shortcuts for working with it.
```

Comment:
See also: https://github.com/scrapinghub/web-poet/issues/210. I think it's fine to document the current behavior though.